test/mandatory_initial_data_removal_test.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:provider/provider.dart';
+import 'common.dart';
+
+void main() {
+  group('Test group', () {
```

Comment:
Could you move those tests next to where FutureProvider/StreamProvider are tested?